modules/luci-mod-dashboard/Makefile

```
@@ -0,0 +1,18 @@
+#
+# Copyright 2019-2020 ZHANG Zhao <Zhao.Zhang2@etu.univ-grenoble-alpes.fr>
+#
+# This is free software, licensed under the Apache License, Version 2.0 .
+#
+
+include $(TOPDIR)/rules.mk
+
+LUCI_TITLE:=LuCI Dashboard Pages
+LUCI_DEPENDS:=+luci-base +libiwinfo +libiwinfo-lua
```

Why is libiwinfo-lua required?

---

I am not very familiar with these dependencies, are there any that I need to remove?

---

I'll look into that.

---

I run tested it and it works without, too. Please remove it.
```suggestion
LUCI_DEPENDS:=+luci-base +libiwinfo
```